web/src/app/admin/connectors/confluence/page.tsx

```diff
@@ -289,15 +295,22 @@ const Main = () => {
               formBody={
                 <>
                   <TextFormField name="wiki_page_url" label="Confluence URL:" />
+                  <BooleanFormField
+                    noPadding
+                    name="index_origin"
+                    label="(For pages) Index the page itself"
+                  />
                 </>
               }
               validationSchema={Yup.object().shape({
                 wiki_page_url: Yup.string().required(
-                  "Please enter any link to your confluence e.g. https://danswer.atlassian.net/wiki/spaces/Engineering/overview"
+                  "Please enter any link to a Confluence space or Page e.g. https://danswer.atlassian.net/wiki/spaces/Engineering/overview"
                 ),
+                index_origin: Yup.boolean(),
               })}
               initialValues={{
                 wiki_page_url: "",
+                index_origin: false,
```

Comment:
prefer default true, that feels like more of the intuitive, expected behavior (e.g. if you specify a page, that it would get indexed)